projects/ids-enterprise-ng/src/lib/version/version-initializer.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+
+// @todo unable to load the version from a file not included as a module in the package.
+// const version = require('../../../../package.json').version;
+// const version = process.env.npm_package_version;
+const version = '5.2.1';
```

Comment:
Seems to be a few suggestions on https://github.com/angular/angular-cli/issues/11931 you could try